### ISSUE TITLE
Durability reset when reloading and there are multiple of the same block type

### ISSUE THREAD

**Issue by Klawdek** [FlyingPikachu/Quests#689]
**What is your Quests version / build number (do _not_ say "latest")?**
3.6.2
**Describe the problem and how to replicate it. Any console errors to post _using Pastebin.com_?**
THe new feature to allow multiple of the same block types does not work.  It shows in the editor as all one kind of block rather than the different types.  The durability is supposed to indicate the different block types and it is reset when questsadmin relad is performed to 0 0 0 0 even though I entered 0 1 2 3.  This causes it to spam the chat with multiple entries and the quest journal to have 7 pages of entries for just 4 logs.
Note last picture.  I had entered 0 1 2 3 fr durability and it was reset to 0 0 0 0 when reload was done.
![screenshot 7795](https://user-images.githubusercontent.com/45950944/52507271-a68fe080-2bb6-11e9-9c57-c163b766e66e.png)
![screenshot 7796](https://user-images.githubusercontent.com/45950944/52507279-adb6ee80-2bb6-11e9-8a0c-33dda93c5ebf.png)
![screenshot 7797](https://user-images.githubusercontent.com/45950944/52507284-b14a7580-2bb6-11e9-988a-c0b1ae9c2f56.png)
![screenshot 7798](https://user-images.githubusercontent.com/45950944/52507286-b3accf80-2bb6-11e9-9a9a-0f91663fe3ff.png)

**If applicable, can you provide an example quest from quests.yml _using Pastebin.com_?**

**Comment by FlyingPikachu** [comment]
Hope to look into this soon. Wish to note that this is not a "new feature" but a removal of the intentional limitation described in #680 in which I warned that such problems could occur.

**Comment by Klawdek** [comment]
Thank you for your response.  Yes you did mention that things could break.  I still think there is a need to be able to have multiple quest objectives that do not have to be done in a particular order.  Like a woodcutter quest.  It is frustrating to be in a birch forest and not be able to get any birch for your quest because what is required at the time is Oak.  That is frustration that in my opinion goes beyond the normal frustration expected in a game.
So if it turns to not be possible the way you are trying it now.  How about a special quest type that does not require the stages to be in a particular order?  Multiple block types could be entered one at a time as different stages.  However the stages would not have to be done in order.   It is not the best solution but it would make the quests plugin more versatile.

**Comment by FlyingPikachu** [comment]
Please try and provide feedback for the following build:
http://ci.ac3-servers.eu/job/Quests/667/

**Comment by Klawdek** [comment]
I tried the new version and the GUI is gone.  When NPC is clicked it prints the quests in the chat.   The multiple logs no longer spam the chat or produce 17 pages of journal entries but it shows all the logs as durability 0 even though the config has other numbers.

**Comment by FlyingPikachu** [comment]
I wasn’t aware that you were using a GUI. Try creating a new one?

Logs will always display like that because Spigot does not provide material names for each durability. You can work around this by setting “translate-items: true” in config.yml

**Comment by Klawdek** [comment]
I tried translate-items: true and it did nothing.  I just did a script reload and server restart.  The gui is still gone.  When I click on NPC it just lists quest in chat.  All the stuff for the gui is in the config. It worked in the previous versions.

**Comment by FlyingPikachu** [comment]
Again, try **creating a new** GUI from the quests editor? If it's been wiped, there's no way to recover it.

Keep `translate-items: true` and try the following build - http://ci.ac3-servers.eu/job/Quests/669/

**Comment by Klawdek** [comment]
version 3.6.4 is still the same.  The items show up as logs in quests journal with no indication as to type of log and the gui is gone.  Apparently it has been wiped and is permanently gone there is now way to get it back.  There is no command for creating a new gui in the editor.  The stuff for the gui is in the editor and in the config it just does not work.  I guess it must wiped as you indicated and permanently gone.  I am sorry to see this feature removed.  I would ask that you consider putting it back in.

**Comment by FlyingPikachu** [comment]
You may need to quit the quest and re-take it (or, if you've already completed the quest, run _/questadmin reset [your name]_), however I've tested that it is in fact working with `translate-items: true`:

![wood](https://user-images.githubusercontent.com/2267126/53280844-4720ed00-36ed-11e9-8dbf-422c9ff66743.png)

Once again, you may create a new GUI to replace the wiped one. The option has not been removed:

![arrow](https://user-images.githubusercontent.com/2267126/53280918-2c02ad00-36ee-11e9-8902-15d9c5be5c86.png)

**Comment by Klawdek** [comment]
I was able to get the gui back by doing the following.  I cut all quests from the file except the default quests.  Then I added a new quests with the quests editor and assigned it a gui item.  Then the gui came back.  I then pasted all my quests back into the file and the gui has been working since. 
While the display in chat when using the /quest command is now correct.  The display in the quest journal still does not show properly.  This is not a real bad problem since the /quest command is easier to use anyway.
